website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -0,0 +1,119 @@
+
+- `TouchableHighlight` - [#31135](https://github.com/facebook/react-native/pull/31135)
+
+- `TouchableOpacity` - [#31108](https://github.com/facebook/react-native/pull/31108)
+
+- `TouchableNativeFeedback` - [#31224](https://github.com/facebook/react-native/pull/31224)
+
+- `TouchableWithoutFeedback` - [#31297](https://github.com/facebook/react-native/pull/31297)
+
+### Selected State Announcement
+
+There were some components that did not announce their selection when in focus. This behavior has now been fixed when the component is in focus and the AccessibilityState is set to selected or the component is changed to selected.
+
+Announces when Selected
+
+- `Button` -  [#31001](https://github.com/facebook/react-native/pull/31001)
+
+- `TextInput` -  [#31144](https://github.com/facebook/react-native/pull/31144)
+
+### Accessibility Timeout Setting
+
+There was previously no way to query the accessibility timeout setting on Android. The fix added the ability to query `AccessibilityManager.getRecommendedTimeoutMillis()`. This queries the "Time to take action" before the UI elements auto-dismisses or auto-progresses.
+
+## Documentation Additions
+
+The React Native documentation must be updated to reflect each addition or change to the available APIs. The [new addition to the React Native documentation](https://reactnative.dev/docs/next/accessibilityinfo#getrecommendedtimeoutmillis-android) covered the addition of `getRecommendedTimeoutMillis()` to AccessibilityInfo.
+
+## Community Involvement
+
+We want to thank all the contributors mentioned below who have submitted and merged pull requests as well as those who have reviewed and commented on issues.
+
+### Merged Pull Requests
+
+- [@huzaifaaak](https://twitter.com/huzaifaaak) closed 3 issues with:
+  - [Added talkback support for button accessibility: disabled prop #31001](https://github.com/facebook/react-native/pull/31001)
+  - [Accessibility/button test #31189](https://github.com/facebook/react-native/pull/31189)
+- [@natural_clar](https://twitter.com/natural_clar) closed 1 issue with:
+  - [feat: set disabled accessibilityState when `TouchableHighlight` is disabled #31135](https://github.com/facebook/react-native/pull/31135)
+- [fabriziobertoglio1987](https://github.com/fabriziobertoglio1987) closed 2 issues with:
+  - [[Android] Selected State does not annonce when `TextInput` Component selected #31144](https://github.com/facebook/react-native/pull/31144)
+  - [Accessibility Fix Image does not announce "disabled" #31252](https://github.com/facebook/react-native/pull/31252)
+- [@kyamashiro73](https://twitter.com/kyamashiro73) closed 1 issue with:
+  - [Added talkback support for `TouchableNativeFeedback` accessibility: disabled prop #31224](https://github.com/facebook/react-native/pull/31224)
+- [grgr-dkrk](https://twitter.com/dkrk0901) closed 1 issue and added to the React Native documentation with:
```

Comment:
Small q: why do certain usernames have `@`? Should it be consistent?

Reply:
I put Twitter handles with an `@` and GitHub handles without.

Reply:
Gotcha! That makes sense! I think `grgr-dkrk` is missing an `@`

Reply:
Thank you for catching that! Fixed